pkg/managerclient/model.go

```diff
@@ -1062,12 +1067,14 @@ func (rp RestoreProgress) Render(w io.Writer) error {
 		}
 	}
 
+	rp.addViewProgress(w)
```

Comment:
Please render view progress after the post-repair progress, so that it follows the overall restore progress.